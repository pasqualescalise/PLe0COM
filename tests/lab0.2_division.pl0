VAR x, y, result : int;

BEGIN
	x := 10;
	y := 5;
	result := x / y;
	print result
END.
