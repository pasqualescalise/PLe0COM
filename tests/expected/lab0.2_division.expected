2
10
122
26
28873036
34
